src/components/detailsHeroSection/detailsHeroSection.module.css

```
@@ -0,0 +1,177 @@
+  gap: 16px;
+  justify-content: center;
+}
+
+@media (max-width: 768px) {
+  .hero {
+    padding: 0;
+  }
+
+  .movieDetail {
+    padding-top: 50px;
+    padding: 16px;
```

Biome (2.1.2)

[error] Unexpected shorthand property padding after padding-top



(lint/suspicious/noShorthandPropertyOverrides)